tests/inventory_test.py

```diff
@@ -31,7 +31,7 @@ def test_create_inventory(test_output_dir):  # noqa: F811
 
     # path = os.path.join(test_output_dir)
 
-    docs_store = DocStore(bucket=path, fs=local_fs, prefix="hazard")
```

Comment:
@joemoorhouse again here we had a quite confusing pattern where, even for a local `DocStore` we had to pass a 'prefix' and a 'bucket'. 

I changed things so that we can pass a `local_path` if `fs` is a local filesystem.